src/ViewRendererBundle/objects/helper/TPkgViewRendererLessCompiler.php

```diff
@@ -81,8 +81,18 @@ protected function getCompiledCssFilename(TdbCmsPortal $portal = null)
         return 'chameleon'.$fileSuffix.'.css';
     }
 
+    /**
+     * @return string - the relative (to PATH_WEB) path pattern to the chameleon css file, without leading slash
+     */
+    public function getCssRoutingPattern(): string
+    {
+        return $this->cssDir.'/'.'chameleon_{portalId}.css';
+    }
+
     /**
      * @return string - the file part for route generation; without a leading slash
+     *
+     * @deprecated since 6.2.4 - use getCssRoutingPattern() which includes the relative path
```

Comment:
Deprecation is only allowed in minor versions, so we need to move this to 6.3.0.

Reply:
The message. Could you file a PR for 6.3 that adds the @deprecated annotation and a notice in the migration guide?